Cargo.toml

```diff
@@ -5,7 +5,7 @@ resolver = "2"
 
 [workspace.package]
 edition = "2021"
-version = "2.2.0"
+version = "2.4.0"
```

Comment:
in anticipation of merging this after https://github.com/NitorCreations/vault/pull/567 which bumps to 2.3